docs/admin_api/README.rst

```diff
@@ -4,17 +4,21 @@ Admin APIs
 This directory includes documentation for the various synapse specific admin
 APIs available.
 
-Only users that are server admins can use these APIs. A user can be marked as a
-server admin by updating the database directly, e.g.:
+Authenticating as a server admin
+--------------------------------
 
-``UPDATE users SET admin = 1 WHERE name = '@foo:bar.com'``
+Many of the API calls in the admin api will require an `access_token` for a
+server admin. (Note that a server admin is distinct from a room admin.)
 
-Restarting may be required for the changes to register.
+A user can be marked as a server admin by updating the database directly, e.g.:
 
-Using an admin access_token
-###########################
+```sql
+UPDATE users SET admin = 1 WHERE name = '@foo:bar.com';
+```
```

Comment:
This is a reStructuredText file, I think the following will work:

```suggestion
.. code-block:: sql

    UPDATE users SET admin = 1 WHERE name = '@foo:bar.com';
```

Reply:
good spot. thanks.